config/src/main/java/com/yahoo/config/subscription/impl/JRTConfigRequester.java

```diff
@@ -194,7 +196,7 @@ private void handleTransientlyFailed(JRTClientConfigRequest jrtReq,
                                          JRTConfigSubscription<ConfigInstance> sub,
                                          long delay,
                                          Connection connection) {
-        transientFailures++;
+        transientFailures = true;
```

Comment:
Should we also set fatalFailures to false here?

Reply:
Yeah, I think that makes sense, I'll do it in the next PR.